factfinder/main.py

```diff
@@ -214,6 +354,12 @@ def aggregate_horizontal(self, client, v, geotype) -> pd.DataFrame:
         )
 
         # Create geoid
+        df = self.create_census_geoid(df, geotype)
+
+        # Output
+        return df[["census_geoid", "pff_variable", "geotype", "e", "m"]]
+
+    def create_census_geoid(self, df:pd.DataFrame, geotype:str) -> pd.DataFrame:
```

Comment:
Should we also convert this from if/elif statements to a lookup? At least this one has a relatively limited number of possible options.

Reply:
yeah we can, it can be a dictionary living in utils